src/Symfony/Component/Messenger/DependencyInjection/MessengerPass.php

```diff
@@ -263,6 +272,9 @@ private function registerReceivers(ContainerBuilder $container, array $busIds)
             foreach ($tags as $tag) {
                 if (isset($tag['alias'])) {
                     $receiverMapping[$tag['alias']] = $receiverMapping[$id];
+                    if (isset($tag['failure_transport']) && null !== $tag['failure_transport']) {
```

Comment:
I suggested renaming this to `is_failure_transport`. Also, it's a boolean, so I think we can do something like:

```suggestion
                    if ($tag['failure_transport']) ?? false) {
```